INSTRUMENTS/SERIAL/SERIAL_SINGLE_MEASUREMENT/SERIAL_SINGLE_MEASUREMENT.py

```diff
@@ -29,8 +29,10 @@ def SERIAL_SINGLE_MEASUREMENT(dc_inputs, params):
     print(",")
     println(reading1)
 
-    If there is more than one column, the SELECT_ARRAY node must be
-    used after this node.
+    This node will receive the data from the arduino as a string and will split
```

Comment:
Can we follow [styleguide](https://github.com/flojoy-io/nodes/blob/main/node_styleguide.md) for docstring?